src/ucp/tag/offload.c

```diff
@@ -556,9 +557,18 @@ ucs_status_t ucp_tag_offload_sw_rndv(uct_pending_req_t *self)
     rndv_rts_hdr = ucs_alloca(rndv_hdr_len);
     packed_len   = ucp_tag_rndv_rts_pack(rndv_rts_hdr, req);
 
-    return uct_ep_tag_rndv_request(ep->uct_eps[req->send.lane],
-                                   req->send.msg_proto.tag.tag,
-                                   rndv_rts_hdr, packed_len, 0);
+    status = uct_ep_tag_rndv_request(ep->uct_eps[req->send.lane],
+                                     req->send.msg_proto.tag.tag,
+                                     rndv_rts_hdr, packed_len, 0);
+    if (ucs_unlikely(status != UCS_OK)) {
+        if (ucs_likely(status == UCS_ERR_NO_RESOURCE)) {
```

Comment:
can you please fix this for `ucp_tag_offload_rndv_zcopy` as well?
Also please check whether any other place with wrong status handling remain

Reply:
can you please fix it here as well?

Reply:
> can you please fix it here as well?

we discussed this with @yosefe and decided to not do it, because they will be replaced by the protocols v2.
see #6287

Reply:
btw, this tag offload which does not support error handling, so even less important

Reply:
@brminich I guess we need to handle the return value for UCT operation anyway - if failure happens, we should gracefully close UCP request and don't crash